packages/core/src/utils/assert.js

```diff
@@ -9,7 +9,6 @@ class PercyAssertionError extends Error {
     'is remote': () => 'Remote resource',
     'no response': () => 'No response',
     'too many bytes': ({ size }) => `Max file size exceeded [${size}]`,
-    'too many redirects': ({ length }) => `Too many redirects [${length}]`,
```

Comment:
This not matter anymore? probably not, I think I remember you mentioning a nice way to pop the final redirect of the request?

Reply:
Other way around actually. In handling the final request, [we can associate the response with the first request in the chain](https://github.com/percy/cli/pull/128/files#diff-47ebb78ad76028830835f9a5e76db43b8cde14c487e756ce6caa82a9a36b369dR174). So any number of redirects should work so long as the browser is fine with it.